docs/quickstarts/ios.mdx

```diff
@@ -0,0 +1,156 @@
+---
+title: Use Clerk with iOS
+description: Learn how to use Clerk to quickly and easily add secure authentication and user management to your iOS application.
+---
+
+# iOS Quickstart (Beta)
+
+<TutorialHero
+  framework="ios"
+  beforeYouStart={[
+    {
+      title: "Set up a Clerk application",
+      link: "https://clerk.com/docs/quickstarts/setup-clerk",
+    },
+  ]}
+>
+
+- Add the Clerk iOS SDK to your application
+- Sign in an existing user
+- Access information about your user
+
+</TutorialHero>
+
+Learn how to use Clerk to quickly and easily add secure authentication and user management to your iOS application.
+
+<Steps>
+
+### Install the SDK
+
+##### Using Swift Package Manager (SPM)
+
+Open the following menu item in Xcode:
+
+File > Add Package Dependencies...
+
+In the Search or Enter Package URL search box enter this URL:
+
+```swift
+https://github.com/clerk/clerk-ios
+```
+
+Then, select your preferred dependency rule and press Add Package.
+
+<Callout>
+For more information on SPM, visit the [official documentation](https://developer.apple.com/documentation/xcode/adding-package-dependencies-to-your-app).
+</Callout>
+
+### Configure and Load Clerk
+
```

Comment:
```suggestion

To configure Clerk in your local project, you first need your publishable API key:

<SignedIn>

<Callout type="info">
  Because you're signed in, the code snippet below will contain the publishable key for your most recently active project. Unless you want to use a different project, you can skip these steps.
</Callout>

</SignedIn>

1. Navigate to the Clerk Dashboard.
2. In the navigation sidebar, select [API Keys](https://dashboard.clerk.com/last-active?path=api-keys).
3. In the **Quick Copy** section, copy your Clerk publishable key.
4. Add the following code snippet to your project, assigning your publishable key to the `publishableKey` property:
```
Adds instructions for getting the API key